CONTRIBUTING.md

```diff
@@ -145,8 +145,11 @@ New versions of SQLFluff will be published to PyPI automatically via
 whenever a new release is published to github. 
 
 A new release can published with a tag in github by navigating to the 
-[releases page](https://github.com/sqlfluff/sqlfluff/releases) and clicking on "Draft a
-New Release" (or clicking [here](https://github.com/sqlfluff/sqlfluff/releases/new)).
+[releases page](https://github.com/sqlfluff/sqlfluff/releases) and a Draft release should
+already exist with merged Pull Requests automatically added since the last release.
+Copy the changes to the [CHANGELOG.md](CHANGELOG.md) file and update the
```

Comment:
Oh, I misinterpreted the mention of CHANGELOG.md earlier. I thought it was the source, but it's the destination.
```suggestion
Copy the text from the draft release into a new, version-numbered section of the [CHANGELOG.md](CHANGELOG.md) file and update the
```